mos-platform/atari8-xegs/syms.s

```diff
@@ -0,0 +1,13 @@
+; xegs ROM size, only 32, 64, 128 and 256 are valid sizes (in KiB)
+
+.weak __cart_rom_size
+; default to 256K
+__cart_rom_size = 256
+
+; explicitly select bank 0, allows for override of INIT
```

Comment:
What about selecting bank zero allows overriding INIT?

Reply:
Poor wording in my part.